### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
There are an ocean 🌊 of misspellings.
Very hard to detect them with `typos` as technical abbreviations look like typos.

### CODE REVIEW

**Comment by szepeviktor** [comment]
`.typos.toml`
```toml
[files]
extend-exclude = [
    ".git/",
    "runtime/tutor",
    "runtime/spell",
    "runtime/lang",
    "runtime/doc",
    "src/po",
    "src/testdir",
    "runtime/syntax",
    "nsis/lang",
]
ignore-hidden = false

[default]
extend-ignore-re = [
    "0x[0-9a-z]{8}u",
    "GROUPE BULL",
    "Christophe Devine",
]
[default.extend-identifiers]
"O_WRONLY" = "O_WRONLY"
"HLF_DED" = "HLF_DED"
"T_ND" = "T_ND"
"t_nd" = "t_nd"
"one_adjust_nodel" = "one_adjust_nodel"
"CMD_enew" = "CMD_enew"
[default.extend-words]
"alloced" = "alloced"
"clen" = "clen"
"windo" = "windo"
"sav" = "sav"
"mak" = "mak"
"dum" = "dum"
"tolen" = "tolen"
"sover" = "sover"
"tpos" = "tpos"
"inex" = "inex"
"fo" = "fo"
"teh" = "teh"
"dout" = "dout"
"ue" = "ue"
"ser" = "ser"
"charcol" = "charcol"
"caf" = "caf"
"wee" = "wee"
"nore" = "nore"
```

**Review by yegappan — APPROVED** [review]

File: runtime/indent/rapid.vim [hunk 1]
```diff
@@ -179,8 +179,8 @@ function s:RapidLenTilStr(lnum, str, startIdx) abort
   return -1
 endfunction
 
-" a:lchar shoud be one of (, ), [, ], { or }
-" returns the number of opening/closing parenthesise which have no
+" a:lchar should be one of (, ), [, ], { or }
+" returns the number of opening/closing parenthesis which have no
```
Comment (dkearns) [inline runtime/indent/rapid.vim]: parentheses - plural form

Comment (szepeviktor) [reply runtime/indent/rapid.vim]: Thank you.

Comment (szepeviktor) [reply runtime/indent/rapid.vim]: Done 🍏

**Comment by szepeviktor** [comment]
@chrisbra The workflow ❌ here is not my fault.

**Comment by chrisbra** [comment]
yes, it's still a bit flaky. no problem

File: src/pty.c [hunk 2]
```diff
@@ -431,7 +431,7 @@ mch_isatty(int fd)
     struct strioctl istr;
 
     istr.ic_cmd = ISPTM;
-    istr.ic_timout = 0;
```
Comment (ObserverOfTime) [inline src/pty.c]: This is *not* a typo. See [`man stropts.h`](https://manned.org/stropts.h.0p).

Comment (szepeviktor) [reply src/pty.c]: Thank you @ObserverOfTime!
You are right. https://docs.oracle.com/cd/E88353_01/html/E37842/stropts.h-3head.html

Comment (szepeviktor) [reply src/pty.c]: Reverted 🍏

**Comment by ObserverOfTime** [comment]
I just noticed a typo that I made. Can you fix it too while you're at it?
```diff
--- i/runtime/syntax/pymanifest.vim
+++ w/runtime/syntax/pymanifest.vim
@@ -20,7 +20,7 @@ syn match pymanifestComment /\\\@1<!#.*/ contains=pymanifestTodo
 " Commands
 syn keyword pymanifestCommand
             \ include exclude
-            \ recursive-include resursive-exclude
+            \ recursive-include recursive-exclude
             \ global-include global-exclude
             \ graft prune
```

**Comment by szepeviktor** [comment]
> I just noticed a typo that I made. Can you fix it too while you're at it?

Of course.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ maillist.  Try reproducing the problem without any of your plugins or settings:
 
 If you report an issue, please describe exactly how to reproduce it.
 For example, don't say "insert some text" but say what you did exactly:
-"ahere is some text&lt;Esc&gt;".
+"here is some text&lt;Esc&gt;".
```
Comment (zeertzjq) [inline CONTRIBUTING.md]: This is also not a typo

**Review by zeertzjq — CHANGES_REQUESTED** [review]

**Comment by zeertzjq** [comment]
I've added the changes to src/ from this PR to #12573. If you remove changes to src/ from this PR then this may no longer need a minor patch.

**Comment by zeertzjq** [comment]
I've added the changes to src/ from this PR to #12753. If you remove changes to src/ from this PR then this may no longer need a minor patch.

File: CONTRIBUTING.md (hunk 3 above)
Comment (szepeviktor) [reply CONTRIBUTING.md]: Reverted.

**Comment by szepeviktor** [comment]
> remove changes to src/

Done 🍏

**Comment by szepeviktor** [comment]
Thank you.

**Pull request closed without merging**